lib/fog/azurerm/requests/network/detach_route_table_from_subnet.rb

```diff
@@ -0,0 +1,53 @@
+module Fog
+  module Network
+    class AzureRM
+      # Real class for Network Request
+      class Real
+        def detach_route_table_from_subnet(resource_group, subnet_name, virtual_network_name, address_prefix, network_security_group_id)
```

Comment:
can we rename these request to just
attach/detach_route_table if the name does not conflict?